core/storage/user/gae_models.py

```diff
@@ -2068,6 +2068,83 @@ def increment_score_for_user(cls, user_id, score_category, increment_by):
             model.put()
 
 
+class UserCommunityRightsModel(base_models.BaseModel):
```

Comment:
/cc @vojtechjelinek for info since he is planning to migrate user IDs.

Reply:
LGTM, all the methods relevant to migration or deletion procedure are correct